lib/query.rb

```diff
@@ -198,6 +198,22 @@ def generate_aggregations(json)
     end
   end
 
+  def set_geo_instance_variables(options)
+    @countries = options[:countries].upcase.split(',').map(&:strip) if options[:countries].present?
+    @trade_regions = split_to_array(options, :trade_regions)
+    @world_regions = split_to_array(options, :world_regions)
+  end
+
+  def split_to_array(options, key)
+    options[key].split(',').map(&:strip) if options[key].present?
+  end
```

Comment:
I don't understand why you pass the entire hash as the first arg and the key you wish to split as the second. Why not `def split_to_array(string)`?